input/pagecontent/reference.md

```diff
@@ -634,6 +634,12 @@ A special case of the ElementDefinition path is setting properties of the first
 . ^<element of ElementDefinition[0]>
 ```
 
+For a path to a code within a code system, use this syntax:
+
+```
+<code in CodeSystem> ^<element of corresponding concept>
```

Comment:
To be consistent w/ how the syntax is described in [3.6.2 Defining Code Systems](http://build.fhir.org/ig/HL7/fhir-shorthand/reference.html#defining-code-systems):
```suggestion
#{code} ^<element of corresponding concept>
```